tests/conftest.py

```diff
@@ -252,159 +252,161 @@ def create_traj(k_shape, nkx, nky, nkz, sx, sy, sz):
 COMMON_MR_TRAJECTORIES = pytest.mark.parametrize(
     ('im_shape', 'k_shape', 'nkx', 'nky', 'nkz', 'sx', 'sy', 'sz', 's0', 's1', 's2'),
     [
-        # (0) 2d cart mri with 1 coil, no oversampling
-        (
-            (1, 1, 1, 96, 128),  # img shape
-            (1, 1, 1, 96, 128),  # k shape
-            (1, 1, 1, 128),  # kx
-            (1, 1, 96, 1),  # ky
-            (1, 1, 1, 1),  # kz
-            'uf',  # kx is uniform
-            'uf',  # ky is uniform
-            'z',  # zero so no Fourier transform is performed along that dimension
-            'uf',  # k0 is uniform
-            'uf',  # k1 is uniform
-            'z',  # k2 is singleton
+        (  # (0) 2d cart mri with 1 coil, no oversampling
+            (1, 1, 1, 96, 128),  # im_shape
+            (1, 1, 1, 96, 128),  # k_shape
+            (1, 1, 1, 128),  # nkx
+            (1, 1, 96, 1),  # nky
+            (1, 1, 1, 1),  # nkz
+            'uniform',  # sx
```

Comment:
I would keep the comments "kx is uniform" and so on because they highlight which of all these parameters determines what.

Reply:
Suggestion: renaming sx and s1 to type_kx and type_k1 etc?
Then the "type_k2=uniform" already explains what is going on?